src/components/SponsorSection/SponsorSection.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+  SponsorContainer,
+  SponsorLogo,
+  SponsorSectionContainer,
+  SponsorText,
+  SponsorWrapper,
+  TitleSponsorContainer,
+  TitleSponsorLogo,
+  TitleSponsorWrapper,
+} from './styles';
+import Sponsors from '../../../config/content/sponsors';
+
+const { titleSponsor, sponsors } = Sponsors;
+const SponsorSection = () => (
+  <SponsorSectionContainer>
```

Comment:
Wrap it inside section layout see FAQ section.

Reply:
@Shurtu-gal I've made the changes. Please review the PR